static/webxdc-preload.js

```diff
@@ -62,6 +68,10 @@
       onStatusUpdate()
       return promise
     },
+    setEphemeralUpdateListener: (cb) => {
+      ipcRenderer.invoke('webxdc.sendGossipAdvertisement')
```

Comment:
maybe we should trigger a native confirmation dialog to inform users that p2p "leaks"/exposes the users ip address to the other group memebers.

Reply:
like "more info"(linking to inApp Help), "Deny", "Allow" and a checkbox "remember choice for this webxdc app"

Reply:
> "leaks"/exposes the users ip address to the other group memebers

To be precise, it's not just that the IP is revealed to the members, but also that whoever is observing your connection can see who you're connecting to, which is more important when talking to people that you trust.